a3p-integration/proposals/a:upgrade-next/auction.test.js

```diff
@@ -0,0 +1,9 @@
+import test from 'ava';
+import { getVatDetails } from '@agoric/synthetic-chain';
+
+test('new auction vat', async t => {
+  const auctionDetails = await getVatDetails('auction');
+  console.log(`AUC `, auctionDetails);
+
+  t.pass('new Auction');
```

Comment:
shouldn't need t.pass(). what can we assert about `auctionDetails`?

Reply:
` getVatDetails('auction')` finds the first match.
`agd query -o json vstorage data  published.agoricNames.instance` shows the current one, but I don't see how to tell that it's the new one.

I can use `pgrep` to verify that there are two auctioneer vats.